test/msw-testing-lib-react/src/utils/test-utils.tsx

```diff
@@ -0,0 +1,14 @@
+import { render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import type { ReactElement } from 'react'
+
+const customRender = (ui: ReactElement, { ...renderOptions } = {}) => {
```

Comment:
Think it might be good if we keep this file equal across examples. Think the one in the other example is a little bit more readable vs the spreading here and further down below.

Reply:
@mitchelvanbever gotcha gotcha, I did some copy paste from previous projects where I used MSW :joy: :fire: